content/develop/start/truffle.md

```diff
@@ -128,22 +128,21 @@ section:
 
 The minter mints and transfers NFT tokens for the vaccine program participant.
 In this example, the new participant address is
-`0x2531a4D108619a20ACeE88C4354a50e9aC48ecfe` and the minter address is
-`0x6A33382de9f73B846878a57500d055B981229ac4`.
+`accounts[1]` and the minter address is `accounts[0]`.
 
 Please make sure that you are using the same deployer address as a minter
 address, otherwise the `mint` transaction will revert.
 
 ```bash
 % truffle console --network aurora
 truffle(aurora)> const cvt = await CovidVaccineToken.deployed()
-truffle(aurora)> await cvt.minter()
-'0x6A33382de9f73B846878a57500d055B981229ac4'
-truffle(aurora)> await cvt.mint('0x2531a4D108619a20ACeE88C4354a50e9aC48ecfe' , {from: accounts[0]})
+truffle(aurora)> await cvt.minter() == accounts[0]
```

Comment:
Would be great if you replace the `accounts[0]` and `accounts[1]` with names that reflect the roles.

```javascript
const minter = accounts[0]
const participant = account[1]
```

Reply:
Good idea! Done in https://github.com/aurora-is-near/doc.aurora.dev/pull/13/commits/089612523e6523ed70dd285493313835a4cd7a8d